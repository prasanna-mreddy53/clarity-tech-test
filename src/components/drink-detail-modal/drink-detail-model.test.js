import React from "react";
import { shallow } from "enzyme";
import DrinkDetailModal from "./drink-detail-modal.component";
import store from "../../reducers/store";
import configureStore from "redux-mock-store";

const setup = () => {
  const component = shallow(<DrinkDetailModal store={store} />);
  return component;
};
const buildStore = configureStore();

describe("Drink Detail Component Tests", () => {
  let drinkDetailModal;
  let store;
  const initialState = {
    drinks: [
      {
        idDrink: "11007",
        strDrink: "Margarita",
      },
    ],
  };
  beforeEach(() => {
    store = buildStore(initialState);
    drinkDetailModal = setup();
  });

  it("renders correctly", () => {
    expect(drinkDetailModal).toMatchSnapshot();
  });
});
