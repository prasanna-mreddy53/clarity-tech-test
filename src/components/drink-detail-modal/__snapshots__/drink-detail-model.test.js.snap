// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drink Detail Component Tests renders correctly 1`] = `ShallowWrapper {}`;
